src/firefly/js/ui/PopupStoreConnection.jsx

```diff
@@ -36,7 +37,7 @@ export class PopupStoreConnection extends PureComponent {
         var {visible}= this.state;
         if (!visible) return false;
         var {dialogId,popupPanel,requestOnTop,zIndex}= this.props;
-        return  React.cloneElement(popupPanel,
+        return  get(popupPanel, 'type.name') !== 'PopupPanel' ? popupPanel : React.cloneElement(popupPanel,
```

Comment:
I don't know why this line is necessary.  You could ignore the properties that are passed. (FYI- The requestToClose, dialogId, visible, property is to support the x, which we might need to add later).